src/Serialization/Combinatorics.jl

```diff
@@ -3,14 +3,14 @@ using JSON
 ###############################################################################
 ## Graphs
 ###############################################################################
-@registerSerializationType(Graph{Directed}, "Graph{Directed}")
-@registerSerializationType(Graph{Undirected}, "Graph{Undirected}")
+@register_serialization_type Graph{Directed} "Graph{Directed}"
+@register_serialization_type Graph{Undirected} "Graph{Undirected}"
```

Comment:
I think this can now be changed, can't it?
```suggestion
@register_serialization_type Graph{Directed}
@register_serialization_type Graph{Undirected}
```
At least I think `string(Graph{Directed})` now returns `"Graph{Directed}"` when used inside Oscar?